content/en/docs/contributing/release-process.md

```diff
@@ -12,16 +12,25 @@ new release of cert-manager.
 
 First ensure that you have all the tools and permissions required to perform a cert-manager release:
 
-1. Install [Kubernetes Release Notes Generator](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md).
-2. Install [cert-manager release tooling](https://github.com/cert-manager/release).
-3. Get permission to use the "cert-manager-release" project in Google Cloud Platform.
+1. Install the [`release-notes`]((https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md)) CLI:
+   ```sh
+   (cd && GO111MODULE=on go get k8s.io/release/cmd/release-notes@latest)
+   ```
```

Comment:
This is what I do, but below you talk about the `krel release-notes` subcommand, which may be the proper way to use these tools.
Make it consistent anyway.